src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -38,11 +38,24 @@ public function prepend(ContainerBuilder $container)
             return;
         }
 
-        if (!isset($frameworkConfiguration['serializer'], $frameworkConfiguration['serializer']['enabled'])) {
+        $serializerConfig = null;
```

Comment:
If I remove the two variables, I have a compiler error, because are used in line 46 and 50.